### PR TITLE
Clean m2 wdl. Closes #4072.

### LINKED ISSUE #4072: GATK4 default jar conventions differ in M2 and CNV WDLs.
The default jar is taken to be the one bundled with the Docker in the CNV WDLs but is an exposed argument in the M2 WDLs.

### PR DESCRIPTION
**Pull request by davidbenjamin**
@LeeTL1220 This consists of a lot of trivial changes where the commit message is self-explanatory, most of which (eg indentation) yield big diffs.

Basically it comes down to:
* M2 wdls have better formatting
* autoval works in the cloud
* gatk launch script and docker jar are used properly in all M2 wdls, including the unsupported ones.

### CODE REVIEW

File: scripts/m2_cromwell_tests/test_m2_wdl_multi.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
-  "Mutect2_Multi.gatk4_jar": "/root/gatk.jar",
+  "Mutect2_Multi.gatk": "/root/gatk.jar",
```
Comment (LeeTL1220) [inline scripts/m2_cromwell_tests/test_m2_wdl_multi.json]: Can't we get rid of the gatk parameter?

File: scripts/m2_cromwell_tests/test_m2_wdl_multi.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+  "Mutect2_Multi.picard": "/home/travis/picard.jar",
+  "Mutect2_Multi.gatk_docker": "__GATK_DOCKER__",
+  "Mutect2_Multi.oncotator_docker": "broadinstitute/oncotator:1.9.3.0",
```
Comment (LeeTL1220) [inline scripts/m2_cromwell_tests/test_m2_wdl_multi.json]: Can you update this?

Comment (LeeTL1220) [reply scripts/m2_cromwell_tests/test_m2_wdl_multi.json]: 1.9.6.1

File: scripts/mutect2_wdl/mutect2_multi_sample.wdl [hunk 3]
```diff
@@ -1,68 +1,34 @@
 #  Run Mutect 2 on a list of tumors or tumor-normal pairs
 #
 #  Description of inputs
-#  gatk4_jar: java jar file containing gatk 4 (protected)
+#  gatk: java jar file containing gatk 4 (protected)
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2_multi_sample.wdl]: This comment seems to be out of date.  Maybe you can delete it even?

File: scripts/m2_cromwell_tests/test_m2_wdl_multi.json (hunk 1 above)
Comment (davidbenjamin) [reply scripts/m2_cromwell_tests/test_m2_wdl_multi.json]: Yep.

File: scripts/m2_cromwell_tests/test_m2_wdl_multi.json (hunk 2 above)
Comment (davidbenjamin) [reply scripts/m2_cromwell_tests/test_m2_wdl_multi.json]: done

File: scripts/mutect2_wdl/mutect2_multi_sample.wdl (hunk 3 above)
Comment (davidbenjamin) [reply scripts/mutect2_wdl/mutect2_multi_sample.wdl]: done

**Comment by davidbenjamin** [comment]
I will punt for now on the template.

**Comment by LeeTL1220** [comment]
@davidbenjamin Feel free to merge once test failures are resolved.

**Pull request merged**